src/features/TakeNow/lib/useTakeNowValidation.tsx

```diff
@@ -23,7 +23,8 @@ export const useTakeNowValidation = ({
   respondentId,
   multiInformantAssessmentId,
 }: TakeNowParams): TakeNowValidatedState => {
-  const [workspaceId, setWorkspaceId] = useState<string | null>();
+  const [fetchSourceSubject, setFetchSourceSubject] = useState<boolean>();
+  const [fetchTargetSubject, setFetchTargetSubject] = useState<boolean>();
```

Comment:
Should these be initialized to false?

Reply:
Thanks Marty. This is part of the bug you observed. I have some logic that depends on this being initially set to `undefined` so I've fixed disabling the query by explicitly comparing to `true`.